packages/reports/src/Stores/Modules/Profit/Helpers/format-response.ts

```diff
@@ -0,0 +1,37 @@
+import { formatMoney, toMoment, getSymbolDisplayName, getMarketInformation } from '@deriv/shared';
+import { ActiveSymbols, ProfitTableResponse } from '@deriv/api-types';
+
+export type TTransaction = NonNullable<NonNullable<ProfitTableResponse['profit_table']>['transactions']>[number];
+
+export const formatProfitTableTransactions = (
+    transaction: TTransaction,
+    currency: string,
+    active_symbols: ActiveSymbols = []
+) => {
+    const format_string = 'DD MMM YYYY HH:mm:ss';
+    const purchase_time = transaction.purchase_time && `${toMoment(+transaction.purchase_time).format(format_string)}`;
+    const purchase_time_unix = transaction.purchase_time;
+    const sell_time = transaction.sell_time && `${toMoment(+transaction.sell_time).format(format_string)}`;
+    const payout = transaction.payout;
+    const sell_price = transaction.sell_price;
+    const buy_price = transaction.buy_price;
```

Comment:
```suggestion
    const payout = transaction.payout ?? NaN;
    const sell_price = transaction.sell_price ?? NaN;
    const buy_price = transaction.buy_price ?? NaN;
```
you have to add `?? NaN` back to keep the existing functionality.
Explanation is here: https://github.com/binary-com/deriv-app/pull/15327/#discussion_r1613103592